src/java.desktop/share/classes/javax/swing/text/AbstractDocument.java

```diff
@@ -147,14 +147,7 @@ protected AbstractDocument(Content data, AttributeContext context) {
 
         if (defaultI18NProperty == null) {
             // determine default setting for i18n support
-            @SuppressWarnings("removal")
-            String o = java.security.AccessController.doPrivileged(
-                new java.security.PrivilegedAction<String>() {
-                    public String run() {
-                        return System.getProperty(I18NProperty);
-                    }
-                }
-            );
+            String o = System.getProperty(I18NProperty);
```

Comment:
I believe the whole handling of `I18NProperty` (this line and the if-else below) can be simplified to Boolean.getBoolean, similar to what has been done in other places in this PR.

Reply:
I noticed but I did not want to change the pattern any more than I was forced to, so I left it.